### DISCUSSION TITLE
Content Security Policy (CSP)

### DISCUSSION THREAD

**Discussion by jossmanbr** [pocketbase/pocketbase#3013] [Q&A]
I have doubts about how the content security policy (CSP) works, or what configurations are necessary to have this security layer when using pocketbase, I hope someone can help me.

**Comment by ganigeorgiev** [comment]
PocketBase comes with default CSP only for the Admin UI and files serving route.

If you use a reverse proxy (eg. nginx, apache, caddy) or have another backend (eg. node) then it is up to you to add the proper CSP. The CSP can be added either as `meta` tag to your html pages or as response header.

For more details, you could check:
- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
- https://content-security-policy.com
